### ISSUE TITLE
пидор дня

### ISSUE THREAD

**Issue by pongo** [pongo/rapturebot#69]
давай в бота добавишь функционал пидора дня, бот из списка кто в чате будет рандомно выбирать и еще чтобы стата была кто сколько раз и еще веселые обои

наступившего дня. просто в полночь сообщает. и тегает пидора.

---

"Такс, такс, такс, кто сегодня пидор дня? Приветстуем, пидор дня - @юзернейм"

---

И нужно ещё, чтобы он периодически напоминал. То есть, этот человек пишет, а ему реплай от бота - кстати, ты пидор. Раза четыре в день. Или два.

---

по команде /pidor показвал кто пидо дня

---

и стату по команды /pidorstats

Топ-10 **пидоров** за текущий год:

1. One_who_makes — *6 раз(а)*
2. EAglee — *4 раз(а)*
3. X_ind_Y — *2 раз(а)*
4. MrPINRED — *2 раз(а)*
5. Helvete666 — *2 раз(а)*
6. AlexStardust — *2 раз(а)*
7. yshmelev — *1 раз(а)*
8. D0RATY — *1 раз(а)*
9. meowinthesky — *1 раз(а)*
10. alesha_e — *1 раз(а)*

Всего участников — *18*

**Comment by arfald20** [comment]
Привет! Подскажи, а этот функционал реализован? Не могу найти концов, есть бот @SublimeBot в нем как раз реализовано почти все, что тут описано. Но может это кто то за тебя сделал в другом боте?

**Comment by pongo** [comment]
Нет, не реализован. Насчет другого бота — скорее всего просто совпадение.

**Comment by arfald20** [comment]
cлишком крутое совпадение) окей спасибо
